near-sdk/src/store/mod.rs

```diff
@@ -1,2 +1,2 @@
 mod lazy;
-pub use lazy::Lazy;
+pub use lazy::{Lazy, LazyOption};
```

Comment:
Sorry, my comment was a bit misleading. It's not crucial that these things are separated, but it's a bit confusing to mix the `impl`s of each in one module and have them connected, when there is no logical connection between the two. I personally think it would be best if `lazy_option` is in its own folder/module so there isn't any confusion, but I don't mind if this change doesn't come in.

Here is what the change would look like to this file:
```suggestion
mod lazy_option;
pub use lazy::Lazy;
pub use lazy_option::LazyOption;
```

Reply:
Oh, that's what you meant. No worries, it's an easy enough change to make